gitdb/db/loose.py

```diff
@@ -204,8 +205,8 @@ def store(self, istream):
                     writer.close()
             # END assure target stream is closed
         except:
-            if tmp_path:
```

Comment:
`tmp_path` could be `None`, so I don't think the `if tmp_path` should be removed.

Reply:
Oh indeed. That's correct. Fixed